### ISSUE TITLE
Arrow navigation for language selector

### ISSUE THREAD

**Issue by Magnusrm** [Altinn/app-frontend-react#3500]
### Description of the bug

When using the dropdown in Altinn forms we have gotten a request from UDIR to have arrow navigation in the dropdown instead of / in addition to tab-navigation.

UDIR interprets this as a WCAG violation.

![Image](https://github.com/user-attachments/assets/9ece9d15-407e-4612-bccb-fc486b6072a3)

### Steps To Reproduce

1. open language selector in f.eks "frontend-test" app
2. tab into menu
3. attempt to navigate using arrow keys
